src/scrcmd_move_tutor.c

```diff
@@ -0,0 +1,335 @@
+#include "global.h"
+#include "assert.h"
+#include "battle_command.h"
+#include "field_player_avatar.h"
+#include "fieldmap.h"
+#include "heap.h"
+#include "message_format.h"
+#include "msgdata.h"
+#include "overlay_01.h"
+#include "party.h"
+#include "pm_string.h"
+#include "pokemon_storage_system.h"
+#include "pokemon.h"
+#include "scrcmd.h"
+#include "script_pokemon_util.h"
+#include "script.h"
+#include "constants/species.h"
+
+// Note: move tutors for Blast Burn, Hydro Cannon, Frenzy Plant
+// and Draco Meteor are handled separately in scr_seq_0948_T30R0601.s
+
+typedef struct TutorMove {
```

Comment:
these seem like they'd be used quite a lot elsewhere, might be best to put them in a header?

Reply:
I'm very sure it's used only in here (seeing as how `sTutorMoves` is static)

Reply:
might be worth naming it something else then, as tutormove can be vague-ish and may be more appropriate elsewhere, ofc I'm a bit unsure on this, since I don't know if it *actually* will be used elsewhere

Reply:
As I've said I'm sure this is the only place where move tutor related code is, so I don't think it will be ambiguous. That said, If you can think of a better name, I'll use it; otherwise I'm sticking with this name.